nototools/noto_lint.py

```diff
@@ -1823,7 +1823,12 @@ def check_accessiblity(cmap):
         all_glyphs = set(font.getGlyphOrder())
         subsetter = subset.Subsetter()
         subsetter.populate(unicodes=cmap.keys())
-        subsetter._closure_glyphs(font)
+        try:
+          subsetter._closure_glyphs(font)
+        except Exception as e:
```

Comment:
Well, I'm generally wary about catch-all exception blocks that don't later raise the exception. I figure it's less likely we'll be surprised by something down the road if we only catch an exception we expect to be raised. But I suppose since we always alert that something was caught this isn't so bad.